jekyll/install.md

```diff
@@ -28,23 +28,3 @@ css_class: install
     </div>
   </div>
 </div>
-
-<section class="background-faded call-to-action">
-  <section class="content text-centered center-banner">
-    <h1 class="section-heading">
-      <i class="far fa-moon fa-2x" aria-hidden="true"></i>
-      Want a nightly build?
```

Comment:
Why delete this?

Reply:
Because it parachutes the reader into detailed information about nightlies with no context on how to use them.

Also, it pulls more detailed information of interest to a much narrower audience -- the existence of the nightlies -- up to a much higher-level decision that is important to the entire audience.

Reply:
In fact, as I reflect on this more, even a majority of the small group who want to know about nightlies won't initially care about browsing the list of compiled nightlies, because at first they will just use `choosenim devel`. In most cases, presenting a user who wants nightlies with an attractive button-link to the list of compiled nightlies at the very top of their mental decision tree will steer them down the wrong path for their needs.

Reply:
I've built the site with your changes, and without this the download section looks quite empty :(

>  with no context on how to use them

Maybe this link can then take users to an another page with that context and explanations how to use them?

Reply:
A majority of users are using nightly builds of Nim. It's important to let people know they exist.

Reply:
Thanks for the feedback -- I agree! Let me know what you think about this latest commit.

I added content on the **Download** menu landing page to explain the available builds and to frame the operating-system choice.

This required a tiny bit of scss work, at which point I became frustrated that the **Download** menu choice pointed to a file called ``install.md`` with css class ``install``. I changed that to ``download.md`` with css class ``download``. (I stayed with the "install" terminology on the os-specific pages because at that point we really do start talking about installation. That terminology transition seems sensible to me.)